tests/floating-point/fft/harness.fil

```diff
@@ -23,49 +23,81 @@ extern "spiral-fft.v" {
     X13: ['G, 'G+1] 32,
     X14: ['G, 'G+1] 32,
     X15: ['G, 'G+1] 32,
+    X16: ['G, 'G+1] 32,
+    X17: ['G, 'G+1] 32,
+    X18: ['G, 'G+1] 32,
+    X19: ['G, 'G+1] 32,
+    X20: ['G, 'G+1] 32,
+    X21: ['G, 'G+1] 32,
+    X22: ['G, 'G+1] 32,
+    X23: ['G, 'G+1] 32,
+    X24: ['G, 'G+1] 32,
+    X25: ['G, 'G+1] 32,
+    X26: ['G, 'G+1] 32,
+    X27: ['G, 'G+1] 32,
+    X28: ['G, 'G+1] 32,
+    X29: ['G, 'G+1] 32,
+    X30: ['G, 'G+1] 32,
+    X31: ['G, 'G+1] 32
```

Comment:
These kinds of interfaces show up in Aetherling too. I would be interesting to come up with some predictable lowering scheme where you can write `X[32]` (bundle syntax) for external components or `main` and have the guarantee that it will generate an interface with names `X0`, `X1`, ...

Reply:
Yeah, that would be super nice! Maybe something to implement as a nice feature for externs.